template/src/screens/SplashScreen.tsx

```diff
@@ -1,12 +1,12 @@
 import React, {useEffect, useState} from 'react'
 import {StyleSheet, View} from 'react-native'
-import ScreenContainer from '../components/ScreenContainer'
-import {deviceWidth, responsiveHeight, colors} from '../themes'
-import {useDispatch} from 'react-redux'
 import CodePush from 'react-native-code-push'
 import Progress from 'react-native-progress'
+import {useDispatch} from 'react-redux'
```

Comment:
In the long term, we should use `useAppDispatch`. You can check it [here](https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type)

Reply:
changed to use useAppDispatch